integration-tests/features/src/attribute_checks.py

```diff
@@ -159,3 +159,37 @@ def get_details_node(context):
     data = context.s3_data
 
     return check_and_get_attribute(data, 'details')
+
+
+def check_cve_value(cve):
+    """Check CVE values in CVE records."""
+    pattern = "CVE-([0-9]{4})-[0-9]{4,}"
+
+    match = re.fullmatch(pattern, cve)
+    assert match is not None, "Improper CVE number %s" % cve
+
+    year = int(re.fullmatch(pattern, cve).group(1))
+    current_year = datetime.datetime.now().year
+
+    # well the lower limit is a bit arbitrary
+    # (according to SRT guys it should be 1999)
+    assert year >= 1999 and year <= current_year
+
+
+def check_cve_value_with_score(cve):
```

Comment:
The function is almost the same as `check_cve_value()`.
The only difference is AFAICT the `pattern`, so maybe you could just add a boolean `has_score` argument to the `check_cve_value()` and choose `pattern` based on the `has_score` ?

Reply:
@jpopelka very good idea, I'll update the code. Thanks!